.changeset/smooth-dogs-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+"@faustwp/core": patch
+---
+
+Toolbar should respect show avatars setting in wp
```

Comment:
Suggestion for changeset. We probably also need to indicate that WPGraphQL version 1.xx is required (don't know version yet, but I asked Jason).

```suggestion
Faust Toolbar will now respect the Show Avatars setting in WordPress.
```